tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/replace_standard_id_in_domain_tables_test.py

```diff
@@ -0,0 +1,252 @@
+       (5, 'hpo_condition_occurrence', 'hpo_dataset', 50, 'test_hpo'),
+       (6, 'hpo_condition_occurrence', 'hpo_dataset', 60, 'test_hpo')] col
+)
+SELECT condition_occurrence_id, src_table_id, src_dataset_id, src_condition_occurrence_id, src_hpo_id FROM w, UNNEST(w.col))
+""")
+
+CONCEPT_DATA_TEMPLATE = JINJA_ENV.from_string("""
+DROP TABLE IF EXISTS `{{project_id}}.{{dataset_id}}.concept`;
+CREATE TABLE `{{project_id}}.{{dataset_id}}.concept`
+AS (
+WITH w AS (
+  SELECT ARRAY<STRUCT<concept_id int64, concept_name string, standard_concept string>>
+      [(319835, 'Congested Heart Failure', 'S'),
+       (45567179, 'Congested Heart Failure', null),
+       (45587397, 'Anaemia complicating pregnancy, childbirth and the puerperium', null),
+       (434701, 'Anemia in mother complicating pregnancy, childbirth AND/OR puerperium', 'S'),
+       (444094, 'Finding related to pregnancy', 'S'), 
+       (40398862, 'Ischemic chest pain', null)] col
+)
+SELECT concept_id, concept_name, standard_concept FROM w, UNNEST(w.col))
+""")
+
+CONCEPT_RELATIONSHIP_DATA_TEMPLATE = JINJA_ENV.from_string("""
+DROP TABLE IF EXISTS `{{project_id}}.{{dataset_id}}.concept_relationship`;
+CREATE TABLE `{{project_id}}.{{dataset_id}}.concept_relationship`
+AS (
+WITH w AS (
+  SELECT ARRAY<STRUCT<concept_id_1 int64, concept_id_2 int64, relationship_id string>>
+      [(319835, 319835, 'Maps to'),
+       (45567179, 319835, 'Maps to'),
+       (45587397, 434701, 'Maps to'),
+       (45587397, 444094, 'Maps to')] col
+)
+SELECT concept_id_1, concept_id_2, relationship_id FROM w, UNNEST(w.col))
+""")
+
+
+class ReplaceWithStandardConceptIdTest(BaseTest.CleaningRulesTestBase):
+
+    @classmethod
+    def setUpClass(cls):
+        print('**************************************************************')
+        print(cls.__name__)
+        print('**************************************************************')
+
+        super().initialize_class_vars()
+
+        # Set the test project identifier
+        cls.project_id = os.environ.get(PROJECT_ID)
+
+        # Set the expected test datasets
+        cls.dataset_id = os.environ.get('COMBINED_DATASET_ID')
+        cls.sandbox_id = cls.dataset_id + '_sandbox'
+
+        cls.rule_instance = ReplaceWithStandardConceptId(
+            cls.project_id, cls.dataset_id, cls.sandbox_id)
+
+        # Generates list of fully qualified table names and their corresponding sandbox table names
+        for table_name in cls.rule_instance.affected_tables:
+            cls.fq_table_names.append(
+                f'{cls.project_id}.{cls.dataset_id}.{table_name}')
+            cls.fq_table_names.append(
+                f'{cls.project_id}.{cls.dataset_id}._mapping_{table_name}')
+            sandbox_table_name = cls.rule_instance.sandbox_table_for(table_name)
+            cls.fq_sandbox_table_names.append(
+                f'{cls.project_id}.{cls.sandbox_id}.{sandbox_table_name}')
```

Comment:
I noticed that after the rule was executed, only the sandbox table for `condition_occurrence` was populated and the others were empty. Should empty sandbox tables be deleted after the rule executes since they are not adding any value?

Reply:
I think `tearDown` in super class `BigQueryTestBase` deletes all `fq_table_names` and `fq_sandbox_table_names`
https://github.com/all-of-us/curation/blob/0377c2774ba2ee682b0c0f4cda09cba340587454/tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/bigquery_tests_base.py#L96

Reply:
I am not referring to tests here

Reply:
This has been addressed by the query we discussed.